src/scss/layout/_navigation.scss

```diff
@@ -51,6 +51,7 @@
     transform: translateY(0);
   }
 
+
```

Comment:
This extra line can be removed, so its consistent with the rest of the file

Reply:
Camilo, thanks for your check, the errors that you report me was already fixed.